Cargo.toml

```diff
@@ -7,18 +7,19 @@ description = "Open Telemetry fox Axum and Tracing"
 
 [dependencies]
 http = "1.1.0"
-opentelemetry = { version = "0.22", default-features = false, features = [
+opentelemetry = { version = "0.24", default-features = false, features = [
   "trace",
 ] }
-tracing-opentelemetry = "0.23"
-opentelemetry-http = { version = "0.11", features = ["hyper", "tokio"] }
-opentelemetry-otlp = { version = "0.15", features = ["http-proto"] }
-opentelemetry-zipkin = { version = "0.20", features = [], optional = true }
-opentelemetry_sdk = { version = "0.22", default-features = false, features = [
+tracing-opentelemetry = "0.25"
+opentelemetry-http = { version = "0.13", features = ["hyper"] }
+opentelemetry-otlp = { version = "0.17", features = ["http-proto"] }
+opentelemetry-zipkin = { version = "0.22", features = [], optional = true }
+opentelemetry_sdk = { version = "0.24", default-features = false, features = [
   "rt-tokio",
 ] }
-opentelemetry-semantic-conventions = "0.14"
-tracing-opentelemetry-instrumentation-sdk = { version = "0.18", features = ["http"] }
+opentelemetry-resource-detectors = "0.3.0"
+opentelemetry-semantic-conventions = "0.16"
+tracing-opentelemetry-instrumentation-sdk = { version = "0.19", features = ["http"] }
```

Comment:
This version works with `tracing-opentelemetry@0.24`, so it's incompatible with `tracing-opentelemetry@0.25`